### ISSUE TITLE
TypeError: unhashable type: 'SubclassAccessors' after upgrade to django 3.2b1

### ISSUE THREAD

**Issue by Mogost** [charettes/django-polymodels#55]
After upgrading to Django 3.2, the manage.py call raises an exception.
I'm not sure django-polymodels is the problem, but it's in the log.

<details>
<summary>Traceback</summary>

```
$ ./manage.py
Traceback (most recent call last):
  File "./manage.py", line 8, in <module>
    execute_from_command_line(sys.argv)
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/core/management/__init__.py", line 395, in execute
    django.setup()
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/apps/config.py", line 301, in import_models
    self.models_module = import_module(models_module_name)
  File "/home/user/.pyenv/versions/3.6.6/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/polymodels/models.py", line 99, in <module>
    class BasePolymorphicModel(models.Model):
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/db/models/base.py", line 161, in __new__
    new_class.add_to_class(obj_name, obj)
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/db/models/base.py", line 326, in add_to_class
    value.contribute_to_class(cls, name)
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/polymodels/models.py", line 58, in contribute_to_class
    class_prepared.connect(self.class_prepared_receiver, weak=False)
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/dispatch/dispatcher.py", line 96, in connect
    if not func_accepts_kwargs(receiver):
  File "/home/user/.pyenv/versions/3.6.6/envs/project/lib/python3.6/site-packages/django/utils/inspect.py", line 45, in func_accepts_kwargs
    p for p in _get_signature(func).parameters.values()
TypeError: unhashable type: 'SubclassAccessors'

```
</details>

May relate to https://github.com/django/django/pull/13151

**Comment by charettes** [comment]
Closing as I cannot reproduce against the latest release candidate against the latest versions of Python 3.6, 3.7, 3.8, and 3.9.

If you can't upgrade to the latest 3.6 I assume https://github.com/django/django/pull/14153 should fix it in 3.2 final.

**Comment by Mogost** [comment]
That's right. I checked with the latest version of Python 3.6, everything works.
I also checked with my previous version of Python, a mentioned patch from Adam backported to 3.2rc1, and I can confirm that it works too.
Thank you for your time.
